master/reference/advanced/etcd-secure.md

```diff
@@ -63,15 +63,9 @@ Alternatively, if you have previously defined/exported your environment
 variables, you can run `sudo` with the `-E` flag to pass in your environment:
 
 ```shell
-sudo -E calicoctl node
+sudo -E calicoctl node run
 ```
 
-Here's a list of commands that must be run as root:
```

Comment:
any reason why we're removing this section? I'm guessing because there's a link to detailed command reference guide below